src/main/java/ca/corefacility/bioinformatics/irida/repositories/remote/resttemplate/SequenceFileMessageConverter.java

```diff
@@ -41,9 +37,8 @@ public SequenceFileMessageConverter(String fileName) {
 	 */
 	@Override
 	public boolean canRead(Class<?> clazz, MediaType mediaType) {
-		logger.trace("Testing converter for class " + clazz.getName() + " and mediatype " + mediaType);
-		if (mediaType != null && MEDIA_TYPE.includes(mediaType)
-				&& clazz.equals(Path.class)) {
+		logger.trace("Testing markdownConverter for class " + clazz.getName() + " and mediatype " + mediaType);
```

Comment:
Is it supposed to be clazz? Or was it just a misspelled variable name?

Reply:
Nope not a misspelled variable, you cannot use `class` because it is a reserved keyword.